modules/user_data/templates/07_cluster_setup.sh.tpl

```diff
@@ -128,18 +129,30 @@ if [ $NODE_DNS == $NODE1 ]; then
     elif [ "$IS_CLUSTER" == 200 ]; then
       echo "Cluster exists"
       break
+    elif [ "$IS_CLUSTER" == 412 ]; then
+      echo "Cluster precondition/s are not met"
```

Comment:
Is there any other information in the response?

Reply:
![image](https://github.com/Ontotext-AD/terraform-aws-graphdb/assets/51083637/de7876b9-4e68-4600-8440-34d59b78c1e2)